packages/api/src/webhooks/cannon.ts

```diff
@@ -222,6 +239,69 @@ export default class WebhookCannon {
     );
   }
 
+  async notifyDisabledWebhook(
+    trigger: messages.WebhookTrigger,
+    params?: any,
+    err?: any
+  ) {
+    try {
+      if (trigger.user.emailValid) {
+        if (
+          !(this.supportAddr && this.sendgridTemplateId && this.sendgridApiKey)
+        ) {
+          console.error(
+            `Webhook Cannon| Unable to send notification email to user, id: ${trigger.id}, streamId: ${trigger.stream?.id}`
+          );
+          console.error(
+            `Sending emails requires supportAddr, sendgridTemplateId, and sendgridApiKey`
+          );
+
+          return;
+        }
+
+        let signature_header = "";
+        if (trigger.webhook.sharedSecret) {
+          signature_header = "-H " + params.headers["Livepeer-Signature"];
```

Comment:
Nice! I think you're missing the header name before the value tho, can use a template string as well
```suggestion
        if (params.headers["Livepeer-Signature"]) {
          signature_header = `-H  Livepeer-Signature: ${params.headers["Livepeer-Signature"]}`;
```

Also, we could have a constant for that `"Livepeer-Signature"` not to repeat it everywhere.

Reply:
My bad! Thanks for noticing